windows.fnl

```diff
@@ -483,15 +472,13 @@
   Takes modal menu table-map
   - Hides any previous display numbers
   - Shows display numbers at top right of each screen
-  - Removes previous monitor items if any were added
-  - Adds monitor items based on currently connected monitors
+  - sets monitor items based on currently connected monitors
```

Comment:
Nit caps

Reply:
Sorry, I think I know what you mean.. but I'm lost as to what "nit" means in this context. Would you mind explaining?

Reply:
means that I am nitpicking

Reply:
As in the PR is fine, but I am trying to fix tiny things.

Reply:
got it. Thank you!